contentful_migrations/migrations/1661197009323--new-compose--add-fields.cjs

```diff
@@ -0,0 +1,64 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+const { withHelpers } = require('@jungvonmatt/contentful-migrations');
+
+/**
+ * Contentful migration
+ * API: https://github.com/contentful/contentful-migration
+ * Editor Interfaces: https://www.contentful.com/developers/docs/extensibility/app-framework/editor-interfaces/
+ */
+
+const RESOURCE_CENTER_PAGE = 'pagePageResourceCenter';
+const LEGACY_COMPOSE_PAGE = 'page';
+module.exports = withHelpers(async function (migration, context, helpers) {
+    const pagePageResourceCenterMigration =
+        migration.editContentType(RESOURCE_CENTER_PAGE);
+
+    // 1. Add fields to our custom page that were previously on the Compose: Page model
+    pagePageResourceCenterMigration
+        .createField('title')
+        .name('Page title')
+        .type('Symbol')
+        .localized(true)
+        .required(false) // change this to True later on, after population
```

Comment:
**question**: it cannot be true now because we have existing content entries that do not include this field?

Reply:
Actually, this came from the whack-a-mole stage of trying to work out what was stopping a migration from being successful. We can add an additional migration now which makes the title field required, and you can test drive that on top of the migrations you've already applied. I'll get on that now.